tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/Controllers/ConfigController.cs

```diff
@@ -0,0 +1,163 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Options;
+
+using Swashbuckle.SwaggerGen.Annotations;
+using System.Net;
+using System.Reflection;
+using Microsoft.Legal.MatterCenter.Web.Common;
+
+using Microsoft.AspNetCore.Hosting;
+
+using Microsoft.WindowsAzure.Storage.Table;
+using System.IO;
+
+#region Matter Namespaces
+using Microsoft.Legal.MatterCenter.Utility;
+using Microsoft.Legal.MatterCenter.Repository;
+using Microsoft.Legal.MatterCenter.Models;
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Authorization;
+using System.Text;
+#endregion
+// For more information on enabling MVC for empty projects, visit http://go.microsoft.com/fwlink/?LinkID=397860
+
+namespace Microsoft.Legal.MatterCenter.Web.Controllers
+{
+    [Authorize(ActiveAuthenticationSchemes = "Bearer")]
+    [Route("api/v1/config")]
+    public class ConfigController : Controller
+    {
+        private ErrorSettings errorSettings;
+        private IMatterCenterServiceFunctions matterCenterServiceFunctions;
+        private IConfigRepository configRepository;
+        private GeneralSettings generalSettings;
+        private ICustomLogger customLogger;
+        private LogTables logTables;
+        private IHostingEnvironment hostingEnvironment;
+        /// <summary>
+        /// Constructor where all the required dependencies are injected
+        /// </summary>
+        /// <param name="errorSettings"></param>
+        /// <param name="generalSettings"></param>
+        /// <param name="configRepository"></param>
+        ///    /// <param name="hostingEnvironment"></param>
+        /// <param name="matterCenterServiceFunctions"></param>
+        public ConfigController(IOptionsMonitor<ErrorSettings> errorSettings,
+            IOptionsMonitor<GeneralSettings> generalSettings,
+            IMatterCenterServiceFunctions matterCenterServiceFunctions,
+            IConfigRepository configRepository,
+            IHostingEnvironment hostingEnvironment
+            )
+        {
+            this.errorSettings = errorSettings.CurrentValue;
+            this.matterCenterServiceFunctions = matterCenterServiceFunctions;
+            this.configRepository = configRepository;
+            this.generalSettings = generalSettings.CurrentValue;
+            this.hostingEnvironment = hostingEnvironment;
+        }
+
+        /// <summary>
+        /// Returns all the entries for Configuring the UI
+        /// </summary>
+        /// <param name="configRequest">Request object for POST</param>   
+        [HttpPost("Get")]
+        [SwaggerResponse(HttpStatusCode.OK)]
+        public async Task<IActionResult> Get([FromBody] DynamicTableEntity configRequest)

```

Comment:
I thing it has to be HttpPost and not HttpGet. Since you are using FormBody, I think you have to use HttpPost. Verify the same